### ISSUE TITLE
Empty window but working when typing local IP+port into a different browser

### ISSUE THREAD

**Issue by R3myG** [ksasso/useR_electron_meet_shiny#2]
Hello! 

I was able to reproduce your tutorial successfully. 
As soon as I changed the app.R file with my Shiny app, I get a blank window opening up and no error messages in the console. 
But if I type http://127.0.0.1:9191/ into a different browser, the app starts immediately and run smoothly.
Any ideas how I can solve this?

**Comment by gerardq675** [comment]
The loading of your app you are using is taking longer than electron takes to set up the html. I have found a workaround which is to change the number after the comma (I changed mine to 15000) in this line of code in main.js. But the timeout could be different for each user, so I'm trying to figure that out.

```
setTimeout( () => {
mainWindow.show()
if(process.platform=MACOS){
mainWindow.reload()
}
loading.hide()
loading.close()
}, 15000)
```

**Comment by petegordon** [comment]
Yep; that can be an issue if your app is taking longer to return than 2000 (2 seconds).  Unless you did what you did and change it to 15000 (15 seconds).   Would be great if we could move these discussions to our primary repository.

https://github.com/ColumbusCollaboratory/electron-quick-start/blob/master/main.js

Thanks!
Pete

**Comment by gerardq675** [comment]
Hi Pete,

Only new to github, I can't see where we could discuss this further? no issues section
